Cargo.toml

```diff
@@ -1,14 +1,17 @@
 [package]
 name = "hamlet"
-version = "0.2.0"
+version = "0.2.1"
 authors = [
     "Wim Looman <wim@nemo157.com>",
     "critiqjo <john.ch.fr@gmail.com>"
 ]
 description = "Html token definitions for stream processing"
 
 readme = "README.md"
+homepage = "https://nemo157.com/hamlet"
 repository = "https://github.com/Nemo157/hamlet"
+documentation = "https://nemo157.com/hamlet/docs/0.2.1/hamlet"
```

Comment:
Yep, I think it's worth it to have a link directly to the versioned documentation on crates.io. It's not much effort to update it when bumping the version.